tethysext/atcore/templates/atcore/resource_workflows/report_workflow_results_view.html

```diff
@@ -0,0 +1,234 @@
+{% extends "atcore/resource_workflows/workflow_results_view.html" %}
+{% load tethys_gizmos static %}
+
+{% block app_content_wrapper_override %}
+  {{ block.super }}
+  <input type="button" value="Print Report" id="btnPrint" class="btn btn-info" style="position:absolute;right:20px;top:70px;"/>
+  <script type="text/javascript">
+    $("#btnPrint").on("click", function() {
+      let curURL = window.location.href;
+      history.replaceState(history.state, '', '/steem-report');
+      window.print();
+      history.replaceState(history.state, '', curURL);
+    });
+  </script>
+{% endblock %}
+
+{% block content_dependent_styles %}
+  {{ block.super }}
+  {% if has_tabular_data %}
+    <style>
+      .map-form {
+        width: 500px;
+        height: 500px;
+        padding-left: 25px;
+        padding-top: 25px;
+      }
+      .tab-data {
+        display: block;
+      }
+    </style>
+  {% endif %}
+  <style>
```

Comment:
This style block is long enough that it should really be moved out into a separate css file.